setup.py

```diff
@@ -213,7 +213,7 @@ def build_dependency(lib_name):
 
 setuptools.setup(
     name="aws_crt",
-    version="0.2.2",
+    version="0.2.1",
```

Comment:
Downgrade?

Reply:
tags now have to match version, I'll kick it up to 2.2 after this pr is merged